bot/rand_select.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python3
+import random
+
+def random_select_weighted_list(ls):
+    return random.choices([k[1] for k in ls], weights = [k[0] for k in ls], k = 1)[0]
```

Comment:
Do we need an entire file for this one function?

Reply:
That was just a quick fix since I use it in `prompt.py` and `typos.py`.

Reply:
In the future, if it's something super small like this (this is just one line of code), it's better to just write it twice. If you're working on something that's small but is going to be used everywhere, it might be better to separate it out (i.e. arguments.py and logger.py).